library/general/src/lib/yast2/group.rb

```diff
@@ -0,0 +1,49 @@
+require "yast2/execute"
+
+module Yast2
```

Comment:
Haven't we agreed on placing all YaST classes under Y2Something namespaces?

Reply:
Let me check it, but I think we agreed that 
Yast is for yast infrastructure and legacy code
Yast2 for code in Yast2.rpm
Y2Prefix for yast module code.

Reply:
Well, we tend to use `Y2Prefix` for code which is specific to an area (no matter in which module is placed). It is some kind of "logic" separation.

For instance, we have `Y2Network`, `Y2Autoinstallation`, `Y2Packager` , and so on. And you can find the `Y2Packager` module in yast2.rpm.